### PR TITLE
first full demo

### PR DESCRIPTION
**Pull request by thebioengineer**
Demo given at todays PHUSE meeting. leaving as draft to incorporate updates and smooth out.

### CODE REVIEW

**Comment by thebioengineer** [comment]
This vignette demo's all available features (except dynamic referencing) currently available in valtools. I think it would be beneficial to move this out of a branch and into develop so it can be kept up to date with the PR's that add new features.

File: vignettes/starting-validated-package.Rmd [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+```
+
+```{r setup, warning=FALSE}
+# remotes::install_github("phuse-org/valtools")
+library(valtools)
+```
+
+## Intro
+
+This vignette steps through creating an R package that is intended to be
```
Comment (elimillera) [inline vignettes/starting-validated-package.Rmd:25]:
```suggestion
This vignette steps through the process of creating an R package that is intended to be 
```

File: vignettes/starting-validated-package.Rmd [hunk 2]
```diff
@@ -0,0 +1,411 @@
+---
+title: "Starting New Validation Package using {valtools}"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Starting New Validation Package}
```
Comment (elimillera) [inline vignettes/starting-validated-package.Rmd:5]: To suppress the rendering error

```suggestion
  %\VignetteIndexEntry{Starting New Validation Package using {valtools}}
```

File: vignettes/starting-validated-package.Rmd [hunk 3]
```diff
@@ -0,0 +1,411 @@
...
+
+```
+
+
+## Requirements
+
+To add the content required for validation, we extended the `usethis` process to
+create a family of "vt_use_*" functions.
+
+`vt_use_req()` creates new requirements. It accepts the name of the requirement,
```
Comment (elimillera) [inline vignettes/starting-validated-package.Rmd:62]:
```suggestion
`vt_use_req()` creates a new requirement. It accepts the name of the requirement, 
```

**Review by elimillera — APPROVED** [review]
Couple of minor changes below, otherwise looks good!

**Pull request merged**